octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/EntityComboBox.java

```diff
@@ -93,6 +93,8 @@ public void mouseExit(MouseEvent mouseEvent) {
     private boolean isLoadingIndicatorEnabled = true;
     private boolean isFilteringEnabled = true;
     
+    public static ArrayList<EntityComboBox> entityComboBoxInstances = new ArrayList<EntityComboBox>();
```

Comment:
Change from ArrayList to List, which is an interface, while ArrayList is an implementation. 
https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java
Check here the first two answers for a better explanation why this is preffered.

Reply:
Forgot to delete that line of code :D